### PR TITLE
Update readme (typos/re-wording)

### PR DESCRIPTION
**Pull request by kinow**
Hi,

Not a native speaker, so feel free to suggest edits/removes of the text changed.

Some changes are not easy to be reviewed in GitHub UI, so I will try to point what changed in comments below.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ DearScholar is used in the following projects:
...
 
 ## Acknowledgement
-DearScholar is developed within the context of the research programme "The creative public servant: Observations, explanations and consequences" with project number 406.18.R8.028, financed by the Dutch Research Council (NWO). The main author wants to express his graditute to Glenn Houtgraaf MSc, Liesbeth Faas MSc, and the ICT Services (Radboud University, Nijmegen) for their advice, testing, and feedback; prof. dr. Sandra van Thiel for her encouragements; and last, but not least, all (pilot) respondents for their effort and feedback during the developmental process.
+DearScholar is developed within the context of the research programme "The creative public servant: Observations, explanations and consequences" with project number 406.18.R8.028, financed by the Dutch Research Council (NWO). The main author wants to express his gratitude to Glenn Houtgraaf MSc, Liesbeth Faas MSc, and the ICT Services (Radboud University, Nijmegen) for their advice, testing, and feedback; prof. dr. Sandra van Thiel for her encouragements; and last, but not least, all (pilot) respondents for their effort and feedback during the developmental process.
```
Comment (kinow) [inline readme.md:42]: /s/graditute/gratitude

File: readme.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ Respondents’ answers are not only send to a server, but also saved in the Dear
...
 
 ### Additional features
-DearScholar has the capacity to send short in-app messages to specific respondents. For example, respondents can be sent thank-you messages to show engagement, small encouragements when respondents have missed a measurement occasions, ask follow up questions, or invited respondents to elaborate on particular answers over the phone or through email. These in-app messages are displayed on a separate message screen. An envelope icon appears in the app’s status bar when new messages have been sent. When respondents have read the message, the researcher is noted.
+DearScholar has the capacity to send short in-app messages to specific respondents. For example, respondents can be sent thank-you messages to show engagement, small encouragements when respondents have missed a measurement occasions, ask follow-up questions, or invited respondents to elaborate on particular answers over the phone or through email. These in-app messages are displayed on a separate message screen. An envelope icon appears in the app’s status bar when new messages have been sent. When respondents have read the message, the researcher is noted.
```
Comment (kinow) [inline readme.md:99]: s/follow up/follow-up

File: readme.md [hunk 3]
```diff
@@ -129,7 +129,7 @@ An *empty* table with the following columns:
 |uname | Respondent's username||
 |token | Respondent's device ID (token) as supplied by Apple or Android on device registration||
 
-*Note*. This table is required to register the device IDs (tokens) in order to send push notifications, settings are stored for a single respondent on each row. Because of security issues, push notification files and configurations are excluded from this public repro. Contact the main author if you want to contribute to enhance the push notification capabilities of DearScholar.
+*Note*. This table is required to register the device IDs (tokens) in order to send push notifications, settings are stored for a single respondent on each row. Because of security issues, push notification files and configurations are excluded from this public repository. Contact the main author if you want to contribute to enhance the push notification capabilities of DearScholar.
```
Comment (kinow) [inline readme.md:132]: Not sure if it was supposed to be `repo` for repository, or `repro` for reproduced (as in reproduced experiment or research?)

File: readme.md [hunk 4]
```diff
@@ -265,7 +265,7 @@ In short, how it works:
...
+When you submit an app to the App Store or Google Play Store for publication, your app will be reviewed. The review procedure can be a black box, hence, I share some experiences with my submissions of DearScholar to the stores. While submissions to Google Play Store have been flawless so far, over time the App Store rejected several builds, probability because DearScholar has been categorized as a *Health and Health Research* app. To get the app published on the App Store, I have to show evidence of ethical approval by a formal ethical committee for the running research projects; information about my relationship with the university where I work and the running research projects; and detailed specifications of the Informed Consent Form that needed to be included in the app. While information about the reasons for the study, information about confidentiality and handling of data, a point of contact for participant questions, and withdrawal procedures were specified already, the App Store reviewers also required information about the benefits and risks for research participants as well as a note about consent procedures for minors.
```
Comment (kinow) [inline readme.md]: `s/probability/probably`, `s/approvement/approval` (I think?), `s/were I work/where I work`

File: readme.md [hunk 5]
```diff
@@ -275,20 +275,20 @@ It is my ambition to automate these steps. Meanwhile, performing the steps manua
...
+After setting up the Apache webserver at least two databases need to be set up in MySQL: one database called ```respondents``` for managing the respondents and registering the device IDs (tokens) in order to send push notifications (i.e., the authentication and registration table) *and* a separate database for each project (with all the other tables) to make it possible to run multiple research projects simultaneously using a single production environment. The tables include variables (in the columns) and specific settings or respondents' data (in the rows). The MySQL database initialisation code for the database, all tables, and the suggested user rights can be found in the [mysql](https://github.com/pmkruyen/dearscholar/blob/master/mysql) file. Details about these tables with all options are provided in the **Settings and options** section on this page.
```
Comment (kinow) [inline readme.md:291]: `s/seperate/separate`

**Comment by pmkruyen** [comment]
Thanks for the corrections Bruno! I accepted all of them.

**Comment by pmkruyen** [comment]
Let me know if they all are really accepted in the readme. This is the first time, someone else opened a pull request :)

File: readme.md (hunk 1 above)
Comment (pmkruyen) [reply readme.md:42]: It is gratitude

File: readme.md (hunk 2 above)
Comment (pmkruyen) [reply readme.md:99]: Yes, thanks

File: readme.md (hunk 3 above)
Comment (pmkruyen) [reply readme.md:132]: Oops, a newby error. It should be `repo` indeed.

**Comment by kinow** [comment]
Hi @pmkruyen ! Thanks for the quick review. Should already be available in the readme of the project :-)

**Pull request merged**